content/components/checkbox.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Checkbox
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=15341%3A47574
+react: https://primer.style/react/Checkbox
+description: A checkbox is a form control for single and multiple selections.
+---
+
+## Usage
+
+Use checkboxes to allow users to select multiple items from a list of individual items, or to mark one individual item as selected.
+
+### States
+
+Checkboxes have static dimensions and three different states: rest (default), focus, and disabled.
+
+<img
+  width="960"
+  alt="An image showing checkboxes with three different states, rest, focus, and disabled."
+  src="https://user-images.githubusercontent.com/586552/204882576-5b698ce7-c471-4f4e-9f33-c861842f5743.png"
+/>
+
+### Selected states
+
+Checkboxes are capable of showing two forms of selection: checked (left) or indeterminate (right).
+
+<img
+  width="960"
+  alt="An image showing checkboxes selected as checked and indeterminate"
+  src="https://user-images.githubusercontent.com/586552/204883614-01743d25-1f7d-4261-b59d-8dd7ceaf29d3.png"
+/>
+
+### Best practices
+
+- An individual checkbox or radio should not have its own validation message or style. For more information, see the [Validation Message](../ui-patterns/forms#validation) section in the Forms documentation.
+
+- An individual checkbox or radio button cannot be marked as required. For more information, see the [Required field indicator](../ui-patterns/forms/#required-field-indicator) in the Forms documentation.
+
+- A selection can be marked as required using a [checkbox group](/design/components/checkbox-group).
+
+## Related links
+
+- [Forms](../ui-patterns/forms#related-components)
```

Comment:
I'm not sure if we need to specifically link to the `#related-components` anchor of that page

Reply:
fixed in latest commit ✔️